docs/1.getting-started/5.third-parties.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Third Parties
+description: Learn how to optimize the performance of third-party resources using built-in composables and components.
+navigation.icon: fluent--people-community-20-filled
+---
+
+Nuxt provides a number of composables and built-in libraries that make it easier, and faster, to load third-party resources in your application.
+
+- useScript: Load any third-party script with server-side rendering support and a proxied API.
+- useStyle: Load any third-party stylesheet using various asset strategies.
+- Third Party Wrappers: Wrapper components and composables that make it easier to include different popular third-parties efficiently.
+
+## How third-parties can impact user experience
+
+Third-parties are external resources included, but not directly controlled, by a site owner to add new functionality to a website. Popular examples of third-parties include analytics, video embeds, maps, and social media integrations. Typically, third-party providers offer code snippets that can be added to the `head` or `body` section of the document.
+
+Adding a single third-party resource to your Nuxt application might not have any noticeable impact on performance, but they can quickly begin to affect user experience if you’re not careful. Many third-parties, especially scripts, can take a relatively long time to download and execute, which can delay user interactivity and block page rendering.
+
+Data from the Chrome User Experience Report shows that Nuxt sites that load more third-party resources have lower [Interaction to Next Paint](https://web.dev/articles/inp) (INP) and [Largest Contentful Paint](https://web.dev/articles/lcp) (LCP) pass rates.
```

Comment:
Can we be more specific about the data (maybe in a footnote to the chart)? What month and year is it from? Was it mobile sites only? Are these origins or URLs?

Reply:
added meta, let me know if this is sufficient 😄

Reply:
Looks good